frontend/src/components/pages/admin/DisplayAssessmentsPage.tsx

```diff
@@ -85,6 +87,7 @@ const DisplayAssessmentsPage = (): React.ReactElement => {
       filteredTests = filteredTests.filter(
         (assessment: AssessmentProperties) => {
           const assessmentProperties = [
+            assessment.id,
```

Comment:
can you refactor lines 214, 220, and 226 to use the enum? also i think the tabs don't show the assessments correctly?

<img width="1210" alt="image" src="https://user-images.githubusercontent.com/69697180/227783974-1b00a199-7fd7-4589-bde1-f3207b2b5f4a.png">

Reply:
oops. should be resolved now!